src/main/java/io/supertokens/inmemorydb/queries/ActiveUsersQueries.java

```diff
@@ -28,6 +28,31 @@ public static int countUsersActiveSince(Start start, long sinceTime) throws SQLE
         });
     }
 
```

Comment:
we should discuss if user_id_mapping needs to be considered when creating totp users cause that may affect the result of countUsersEnabledTotpAndActiveSince.

For example, since getUserLastActiveTable() only contains the supertokens user id (and never external userid), if the totp user is equal to the external user id, then the intersection of the two tables will be 0, even though they actually point to the same user.

Reply:
I think we should only store supertokens user id in TOTP. Totp device/user registration doesn't do any checks or user id translation at the moment. It just happily accepts whatever is given.